core/modules/field_ui/field_ui.theme.inc

```diff
@@ -139,11 +142,14 @@ function theme_field_ui_view_modes($variables) {
     $row_classes = array('view-mode--' . str_replace('_', '-', $view_mode_name));
     $row = array();
 
-    $row['label'] = check_plain(t($view_mode_info['label']));
-    $row['machine'] = $view_mode_name;
+    $row['label'] = theme('label_machine_name__node_display__' . backdrop_strtolower($element['#bundle']), array(
```

Comment:
okay, what about this one? ... Isn't $element['#bundle'] the machine name of the bundle? If not, shouldn't we be using that instead of the bundle label?

Reply:
@jenlampton Ok, yes. Took some doing, but I managed to work out how to get the node type machine name (it wasn't in the data provided to this function). PR updated.